eam-light-backendweb/src/main/java/ch/cern/cmms/eamlightweb/workorders/myworkorders/MyWorkOrders.java

```diff
@@ -93,8 +93,9 @@
         // use star organization to get all work orders, independently of organization
         workOrdersGridRequest.addParam("parameter.objorganization", "*");
 
-        workOrdersGridRequest.addFilter("eventtype", "JOB", "=", GridRequestFilter.JOINER.OR);
-        workOrdersGridRequest.addFilter("eventtype", "PPM", "=");
+        workOrdersGridRequest.addFilter("eventtype", "JOB", "=", GridRequestFilter.JOINER.OR, true, false);
+        workOrdersGridRequest.addFilter("eventtype", "PPM", "=", GridRequestFilter.JOINER.AND , false, true);
+        workOrdersGridRequest.addFilter("wotype", "IS - ", "NOTCONTAINS");
```

Comment:
Why contains, and why 'IS - '?

Reply:
'wotype' corresponds to R5EVENTS.EVT_JOBTYPE, including the description of that code (in French in CERN's case), separated by a '-'. I could not find a column in this grid request for the EVT_JOBTYPE column, without the description.

In this case, we wish to filter out the work orders that have the IS job type. Thus, and this is a workaround, the code filters for work orders that do not contain 'IS - ' in the 'wotype' column.

Reply:
Then couldn't it be a BEGINS?

Reply:
`NOTBEGINS` would work, but it must not begin with 'IS - ', which the NOTCONTAINS handles ok.